deltacat/io/object_store.py

```diff
@@ -43,7 +43,7 @@ def get_many(self, refs: List[Any], *args, **kwargs) -> List[object]:
         or may not return ordered results.
         """
 
-    def clear(self, *args, **kwargs) -> bool:
+    def clear(self, *args, **kwargs) -> None:
```

Comment:
Why changed the signature? You can make it Optional[bool] to extend.

Reply:
This is generally fine if `noreply=True`. Since we're explicitly waiting for the ACK here, we should also capture the status of the flush calls.